src/Commands/RunScan.ts

```diff
@@ -219,4 +227,67 @@ export class RunScan implements CommandModule {
       process.exit(1);
     }
   }
+
+  private async validateParams(
+    args: Arguments,
+    scanManager: Scans
+  ): Promise<void> {
+    const projectId = args.project as string;
+    if (!projectId) {
+      logger.error(
+        'Please specify a project ID with the --project (-p) option to run a scan with entrypoints using the --entrypoint option.\n\nExamples:\nbright scan:run --project <project_id> --entrypoint\nbright scan:run --project <project_id> --entrypoint <entry_point_id1> <entry_point_id2> <entry_point_id3>'
+      );
+      process.exit(1);
+    }
+
+    const isProjectExist = await scanManager.isProjectExist(projectId);
+    if (!isProjectExist) {
+      logger.error(
+        `The specified project ID ${projectId} does not exist.\nPlease verify the project ID and try again.`
```

Comment:
This error message can be implemented on `POST /api/v1/scans` thus avoiding extra call to BE